update_actions.py

```diff
@@ -0,0 +1,59 @@
+"""
+  Scans directories for files that satisfy a specific mask and updates
+  the actions' tags.
+  python3 update_actions.py --source “one/*.y*ml two/*.y*ml three/*.y*ml”
+"""
+
+import glob
+import argparse
+import re
+from pathlib import Path
+from github import Github
+
+parser = argparse.ArgumentParser()
+cache = {}
+
+parser.add_argument(
+    '-s', '--sources',
+    type=str, help='Source directories to scan'
+)
+
+args = parser.parse_args()
+sources = args.sources.split(' ')
+g = Github()
+pattern = r"\s+action:.*"
+
+for source in sources:
+    path = Path(source)
+    for file_path in glob.iglob(source):
+        n_actions = 0
+        n_changed = 0
+        with open(file_path) as file:
+            s = file.read()
```

Comment:
Let's make variable names more self-explanatory. Please rename `s`, `p`, `ss`, `g` to a more readable names.